scaleway/data_source_iam_group_test.go

```diff
@@ -0,0 +1,101 @@
+package scaleway
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccScalewayDataSourceIamGroup_Basic(t *testing.T) {
+	SkipBetaTest(t)
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: tt.ProviderFactories,
+		CheckDestroy: resource.ComposeTestCheckFunc(
+			testAccCheckScalewayIamGroupDestroy(tt),
+		),
+		Steps: []resource.TestStep{
+			{
+				Config: `
+					resource "scaleway_iam_group" "main_ds_basic" {
+						name        = "test_data_source_basic"
+					}
+			
+					data "scaleway_iam_group" "find_by_id_basic" {
+						group_id 	= scaleway_iam_group.main_ds_basic.id
+						organization_id = "08555df8-bb26-43bc-b749-1b98c5d02343"
```

Comment:
Can the organization_id be deduced from the provider settings? it seems cumbersome to specify it again

Reply:
For now it is not possible for requests that use query arguments like ListGroups, so it affects the sweeper functions and the search by name in datasources. After discussing it with the team, the solution we found was to temporarily make the organization_id required in datasources (every iam datasource is concerned).
Apparently, it's related to [this issue](https://github.com/scaleway/terraform-provider-scaleway/issues/1337).